src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -430,7 +430,7 @@ public String getAudioCodec() {
 		if (is3GA()) {
 			return "3GA";
 		} else if (isAACLC()) {
-			return "AAC";
+			return "AAC-LC";
 		} else if (isAC3()) {
```

Comment:
It's not safe to change things like this, there's no way to know how many other places in the code this value is hard-coded. If you change it, ```FormatConfiguration``` should be used instead because it's safer.